flask/src/main/groovy/net/corda/gradle/flask/FlaskPlugin.groovy

```diff
@@ -0,0 +1,69 @@
+package net.corda.gradle.flask
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.file.Directory
+import org.gradle.api.plugins.JavaApplication
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.api.plugins.JavaPluginConvention
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Copy
+import org.gradle.api.tasks.JavaExec
+import org.gradle.api.tasks.SourceSet
+import org.gradle.jvm.tasks.Jar
+
+class FlaskPlugin implements Plugin<Project> {
+
+    private static final String GROUP = "Flask"
```

Comment:
Better to declare this as `@PackageScope`, and then you can also use it to set a group for the `FlaskJarTask`. And a description too, of course :grin:.